hierarchicalforecast/methods.py

```diff
@@ -336,6 +335,25 @@ def crossprod(x):
     return x.T @ x
 
 # %% ../nbs/methods.ipynb 33
+def cov2corr(cov, return_std=False):
+    """ convert covariance matrix to correlation matrix
+
+    **Parameters:**<br>
+    `cov`: array_like, 2d covariance matrix.<br>
+    `return_std`: bool=False, if True returned std.<br>
+
+    **Returns:**<br>
+    `corr`: ndarray (subclass) correlation matrix
+    """
+    cov = np.asanyarray(cov)
+    std_ = np.sqrt(np.diag(cov))
+    corr = cov / np.outer(std_, std_)
+    if return_std:
+        return corr, std_
+    else:
+        return corr
+
+# %% ../nbs/methods.ipynb 34
```

Comment:
Cool. Is it necessary to include the `return_std` argument? I did not see it being used in any function.

Reply:
Nice catch, used it now.